server.py

```diff
@@ -641,6 +659,9 @@ def save_reports():
     scaffold_method = request.args.get("scaffold_method")
     model = request.args.get("model")
 
+    # Submit reports to AVID
+    submit_reports_to_AVID(reports)
```

Comment:
One note is that unfortunately, since this function gets called any time a user saves _any_ of their reports, we'll end up with duplicates in the DB if they've saved another report previously. Given the different DB approach, we probably need to modify the `save_reports` function into a `save_report` function that only saves one report (the one that's currently open for the user). 

As another improvement, we may also want to check whether a DB entry already exists so we could update its contents rather than push a new DB entry. It might not be necessary to implement these fixes in this first version, but just wanted to note this!